build.sbt

```diff
@@ -4,6 +4,7 @@ import akka._
 inThisBuild(Def.settings(
   organization := "com.typesafe.akka",
   organizationName := "Lightbend",
+  crossScalaVersions := Seq("2.11.8", "2.12.0"),
```

Comment:
Note, `crossScalaVersions` is currently defined in [`project/Dependencies.scala`](https://github.com/akka/akka-http/blob/master/project/Dependencies.scala#L19) and added via `Dependencies.Versions` below. So if you want to define it here then I suggest the one in `Dependencies.Versions` is removed.